pkgs/development/tools/build-managers/gn/setup-hook.sh

```diff
@@ -1,14 +1,19 @@
+# shellcheck shell=bash disable=SC2164
```

Comment:
## [SC2164](https://www.shellcheck.net/wiki/SC2164) Use `cd ... || exit` in case `cd` fails.

I'm assuming this is triggered by line 12. Could we land the disable on that line?